config.json

```diff
@@ -26,6 +26,37 @@
     "practice": []
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Strong, dynamic typing",
+      "content": "Gdscript is strong and dynamically typed and supports gradual typing.",
+      "icon": "dynamically-typed"
+    },
+    {
+      "title": "Cross platform",
+      "content": "Gdscript code can run on Windows, Linux, macOS, iOS, Android, and the web.",
+      "icon": "cross-platform"
+    },
+    {
+      "title": "C++ bindings",
+      "content": "C++ bindings allows for fast, low-level access to the engine's features.",
+      "icon": "fast"
+    },
```

Comment:
I agree with @pfertyk that this isn't GDScript related, but is Godot related.

GDScript is only the scripting engine that runs on top of the Godot features.